### PR TITLE
Search engine followup and recommendations

### PR DESCRIPTION
**Pull request by davidfischer**
This is a followup with more recommendations and commentary than just running the tools like in the [previous post](https://www.ethicalads.io/blog/2022/10/privacy-teardown-search-engines/). Since none of these search engines are "bad", the recommendations are mostly based on what people are looking for rather than recommending a single one.

Screenshot 2022-11-08 at 16-02-24 Best Search Engines for your Privacy

### CODE REVIEW

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+You don't have to use a search engine like Google which retains your queries forever to get great results anymore.
+Last month, we did a [privacy teardown]({filename}../posts/privacy-teardown-search-engines.md) of some large search engines as well as the more privacy minded ones.
+Based on our results, we're prepared to make a few recommendations and offer a few tips to keep you safe.
+
+
+## Privacy search engine landscape
+
+There's a lot of privacy concerns when searching online.
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:14]:
```suggestion
There's a lot of privacy concerns when searching online:
```

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+It's just a few clicks away to make it your browser's default search engine,
+and when the results aren't quite what you're looking for, you can prefix your query with `!g` to search Google.
+
+The user experience is fantastic and they have mobile apps for Android and iOS as well as a Mac desktop app.
+The mobile app on Android can also double as tracker protection even for requests made in other apps.
+By using Bing's search index under the hood, the search results are pretty close to Google's in quality.
+DuckDuckGo is a great choice for a private search engine.
+
+
+### [Mojeek](https://www.mojeek.com/)
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:42]: No image for them?

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+One big advantage that comes from these integrations is that it can function as a great developer search engine
+and that's why the team built YouCode search.
+YouCode integrates results from common developer sites like StackOverflow
+and Read the Docs, the parent of EthicalAds, directly onto the search engine results screen.
+
+While I couldn't find details on You.com's search index, based on their search results,
+it's clear that at least in some queries You.com is relying on Bing's index.
+That means that the result quality will stay high.
+In our privacy teardown, You.com and YouCode got close to top marks.
+For a search engine tuned for developer queries, You.com and specifically YouCode is a good choice.
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:73]: We should be bolding a primary takeaway from each of these sections to make it skimmable.

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+In our teardown, its results were perfect with no cookies, trackers, third-parties, or any other privacy violations.
+Like Mojeek but unlike the rest of the bunch, Brave has built their own search index
+although in some cases they claim to mix their results with Bing's.
+Despite mostly relying on its own search index, I was pretty impressed with the search result quality and order
+even when compared to Bing itself.
+
+Like DuckDuckGo, Brave has desktop and mobile apps so you can search privately everywhere.
+They claim not to store searches, clicks, or any other data and allow truly anonymous searches.
+Brave specifically mentions that they do not filter or downrank any search results.
+For a good good alternative from a team focused on privacy, you can't go wrong with Brave Search.
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:93]: 2x good

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+* **[Qwant](https://www.qwant.com/)** - Qwant is a French search engine that also uses Bing's search index.
+* **[Searx](https://searx.info/)** - Searx is an open source meta search engine that relies on results from other search engines.
+
+
+## Recommendations
+
+Unless you have specific concerns,
+you can't go too far wrong with any of these privacy-minded alternatives.
+As an every day search engine, DuckDuckGo is popular with the EthicalAds and Read the Docs teams
+while respecting your privacy.
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:116]:
```suggestion
while respecting their privacy.
```

**Review by ericholscher — APPROVED** [review]
This looks great. I do think it could be more scannable and have key points highlighted in bold as I mentioned. Overall it seems like we don't have a strong winner, so we need to be really explicit about why someone might choose each. 

I would love to find a way to break this content up and reuse it somehow, but not sure the best approach. I'll be curious if we get any SEO traffic from the title.

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
+Title: Best Search Engines for your Privacy
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:1]: The filename should probably match this, or at least the URL

File: content/posts/2022-which-search-engine-should-you-use.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+While the search engine results page is a bit spartan,
+it loads very fast and the actual result quality was pretty good.
+If you're looking for an independent search engine that doesn't rely on big tech at all,
+Mojeek is amazing.
+
+
+### [You.com](https://you.com/code)
+
+<div class="postimage text-center">
```
Comment (ericholscher) [inline content/posts/2022-which-search-engine-should-you-use.md:56]: We're swapping whether we put the image above or below the name, we should be consistent with 1.

File: content/posts/2022-which-search-engine-should-you-use.md (hunk 5 above)
Comment (davidfischer) [reply content/posts/2022-which-search-engine-should-you-use.md:116]: Our maybe? Their seems wrong. Might even just drop it entirely.

**Review by ericholscher — COMMENTED** [review]
Changes look great 💯

**Pull request merged**